### PR TITLE
fixes #22085 - facet test models have db table

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

File: test/unit/facet_configuration_test.rb [hunk 1]
```diff
@@ -7,15 +7,14 @@ module TestExtension
 end
 
 class FacetConfigurationTest < ActiveSupport::TestCase
-  class TestClass < HostFacets::Base
+  class TestFacet < HostFacets::Base
```
Comment (ShimShtein) [inline test/unit/facet_configuration_test.rb:10]: There is easier way to solve this:
``` ruby
  class TestClass < HostFacets::Base
    self.table_name = 'test_facets'
  end
```

This sets proper table name for the model hence eliminating non-existing table error.

**Review by ShimShtein — CHANGES_REQUESTED** [review]
@timogoebel, your solution is valid, but we could do it with less code lines. I don't have a strong opinion against it though.

File: test/unit/facet_configuration_test.rb (hunk 1 above)
Comment (timogoebel) [reply test/unit/facet_configuration_test.rb:10]: > your solution is valid, but we could do it with less code lines.

Indeed, but I don't think it's the straight forward solution. The diff is larger, but we don't add more LOC with my proposal. And it's easier to read the code with my change imho.

**Review by ShimShtein — APPROVED** [review]

**Comment by ShimShtein** [comment]
Merged.

Thanks @timogoebel! 
As I said, the solutions are equivalent for me.

**Pull request merged**